### PR TITLE
add authors file

### PR DESCRIPTION
**Pull request by ghost**
see discussion in https://forum.syncthing.net/t/android-transition/11383/14

### CODE REVIEW

File: AUTHORS [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+Cristian Beskid <schizosfera@gmail.com>
+Faisal Hameed <faisal.hameed@devfactory.com>
+Felix Ableitner <me@nutomic.com>
+FlashSystems <developer@flashsystems.de>
+George Venios <veniosg@gmail.com>
+Jakob Borg (calmh) <jakob@nym.se> <jakob@kastelo.net>
+Jessie Chatham Spencer <isai@protonmail.com>
+JohnVeness <john.veness.github@pelago.org.uk>
+kDot <k-Laus@gmx.net>
+klaus <k-Laus@gmx.net>
```
Comment (AudriusButkevicius) [inline AUTHORS]: There is a ton of duplicates with same email.
Also, I see no purpose of this, just downsides of having to maintain this pointless file

Comment (Catfriend1) [reply AUTHORS]: -- deleted as discussion is going on in the forum --

**Comment by Catfriend1** [comment]
Lgtm one more file doesnt really hurt but makes it easy to see the credits. From my opinion it must not be an app dialog according to the latest forum discussion.

**Comment by AudriusButkevicius** [comment]
Sorry, but I see no value in having to maintain this file, the list is already there in some sensible order:

image

**Comment by Catfriend1** [comment]
Closed.

**Pull request closed without merging**